perf/evp_fetch.c

```diff
@@ -223,8 +224,8 @@ int main(int argc, char *argv[])
     }
 
     threadcount = atoi(argv[argnext]);
-    if (threadcount < 1) {
-        printf("threadcount must be > 0\n");
+    if (threadcount < 1 || NUM_CALLS_PER_TEST % threadcount != 0) {
```

Comment:
I would actually prefer a small tweak here:
```
num_calls_per_test = NUM_CALLS_PER_TEST +  (NUM_CALLS_PER_TEST % threadcount)
```
the idea is to adjust the number of iterations so the job can be always evenly divided. This will allow us to use thread counts 32, 64, and similar. This should implemented for all performance tools.

thanks.

Reply:
Makes sense, I'll add it.

Reply:
My suggestion is wrong here, my math is weak. Please go for solid suggestion from @vdukhovni mentioned [here](https://github.com/openssl/tools/pull/196#discussion_r1642659934)
```
/* Assuming `threadcount` is always positive */
int excess = NUM_CTX_PER_RUN % threadcount;
num_ctx_per_run = NUM_CTX_PER_RUN + (excess ? threadcount - excess ? 0);
```